ENVIRONMENTVARS.md

```diff
@@ -0,0 +1,54 @@
+| `NYPL_HEADER_URL` | string | "https://ds-header.nypl.org" | The base URL of the NYPL envionment-specific header and footer scripts.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+
+## AWS ECS Environment Variables
+
+As previously mentioned in the [README](README.md), we are using environment variables to make authorized requests to NYPL's API platform. In order to be secure, we are encrypting and decrypting those environment variables using AWS KMS. Please get these variables from someone on the LSP team.
+
+| Variable | Description |
+| -------- | ----------- |
+| `PLATFORM_API_CLIENT_ID` | Platform client id. This value must be encrypted. |
+| `PLATFORM_API_CLIENT_SECRET` | Platform client secret. This value must be encrypted. |
+
+### Encrypting
+
+`PLATFORM_API_CLIENT_ID` and `PLATFORM_API_CLIENT_SECRET` are assumed to be encrypted. We need these variables to create an instance of the `nypl-data-api-client` npm package and make authorized requests to the NYPL Digital API endpoints. This is needed for the Discovery UI app to make requests itself to the APIs.
+
+In order to encrypt, please download and install the `aws` [cli tool](https://aws.amazon.com/cli/). The command to encrypt is
+
+    aws kms encrypt --key-id [your IAM key from AWS] --plaintext [value to encrypt] --output text --query CiphertextBlob
+
+The `aws kms encrypt` commands returns and object with a `CiphertextBlob` property. Since we only want that value, we use the `--query` flag to retrieve just that. This value can be copied and pasted into the AWS ECS configuration in the UI for the app's environment.
+
+More information can be found in the [encrypt docs](http://docs.aws.amazon.com/cli/latest/reference/kms/encrypt.html).
+
+Alternatively, you can use the [kms-util](https://github.com/NYPL-discovery/kms-util) helper package."
```

Comment:
Minor: extra `"` at the end.

Reply:
Not for right now but this should be expanded with more information on `kms-util` and an example usage. We're saying alternatively that this package can be used but not why or how or differences between the `aws kms ...` script.